XMLHttpRequest/responsedocument-decoding.htm

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>XMLHttpRequest: response document decoding</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <link rel="help" href="https://xhr.spec.whatwg.org/#the-responsetext-attribute" data-tested-assertations="following::ol[1]/li[4]" />
```

Comment:
Should this be changed to https://xhr.spec.whatwg.org/#the-response-attribute ?

Reply:
We should probably just drop these or link without anchor.

Reply:
Removed link elements.